apps/backoffice-v2/src/pages/Entity/hooks/useTasks/hooks/useAssosciatedCompaniesBlock/useAssociatedCompaniesBlock.stories.tsx

```diff
@@ -3,9 +3,9 @@ import { Case } from '@/pages/Entity/components/Case/Case';
 import { Card } from '@/common/components/atoms/Card/Card';
 import { ctw } from '@/common/utils/ctw/ctw';
 import { CardContent } from '@/common/components/atoms/Card/Card.Content';
-import { cells } from '@/pages/Entity/hooks/useEntity/cells';
 import { useAssociatedCompaniesBlock } from '@/pages/Entity/hooks/useTasks/hooks/useAssosciatedCompaniesBlock/useAssociatedCompaniesBlock';
 import { FunctionComponent } from 'react';
+import { cells } from '../../../useEntity/cells';
```

Comment:
Now it doesn't even make sense. Previous file changed to '@' notation, here to dot notation